help_to_heat/frontdoor/views.py

```diff
@@ -621,9 +621,15 @@ def save_post_data(self, data, session_id, page_name):
         country = session_data.get(country_field)
         building_name_or_number = data.get(address_building_name_or_number_field)
         postcode = data.get(address_postcode_field)
```

Comment:
Life easier if you
```suggestion
        building_name_or_number = data.get(address_building_name_or_number_field).strip()
        postcode = data.get(address_postcode_field).strip()
```
up here?

Reply:
perhaps? though would lead to the line
```py
data[address_building_name_or_number_field] = building_name_or_number
```
below, which seems a bit unclear

Reply:
Why's that unclear? Don't think it's any stranger than what we're doing at the moment.

Does trigger the question of whether there's an earlier point we could be doing this, before they get attached to the data object? Assuming not?

Reply:
> Why's that unclear? Don't think it's any stranger than what we're doing at the moment.

just that the line `data[postcode] = postcode` may not appear to do anything useful at first. suppose can add a comment to make this clear

> Does trigger the question of whether there's an earlier point we could be doing this, before they get attached to the data object?

not really; we can specify a preprocessor for the answer schema itself but its of not much help here since this code runs before the schema is used

Reply:
Yeah I think it makes sense: we take the data, format it, and write it back.